### DISCUSSION TITLE
the Extended Address does matched in LLA

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#8803] [Q&A]
Hi all,
wht is The LLA not same as the leader's Extended Address? 
![image](https://user-images.githubusercontent.com/30258768/221120600-095a428a-c447-464a-b694-2979d0b2721d.png)

**Comment by wgtdkp** [comment]
I think it's the Extended PAN ID (extpanid) but not the Extended Address (extaddr). Use OT command `extaddr` to get the Extended Address of the device.

**Reply by Leo2442926161** [reply]
Thank you Wgtdkp,
Seems still not same.
![image](https://user-images.githubusercontent.com/30258768/221129029-2cc05596-235e-4c9f-8ab9-caea3c7157a9.png)

**Reply by jwhui** [reply]
IPv6 link-local address should be derived from IEEE 802.15.4 Extended Address on the same device. What platform are you using?

**Reply by Leo2442926161** [reply]
Thanks Jwhui.

**Reply by Leo2442926161** [reply]
when will the extaddr will change?

**Reply by jwhui** [reply]
The IEEE 802.15.4 Extended Address is:

1. Initialized to a random value on startup:
https://github.com/openthread/openthread/blob/27802bc0df22480e1a76ddb82fa336b006dddd53/src/core/mac/mac.cpp#L126

2. Restored to a saved value when reattaching to a network:
https://github.com/openthread/openthread/blob/27802bc0df22480e1a76ddb82fa336b006dddd53/src/core/thread/mle.cpp#L400

3. Set to the Joiner ID when operating as a Joiner during Thread Commissioning:
https://github.com/openthread/openthread/blob/27802bc0df22480e1a76ddb82fa336b006dddd53/src/core/meshcop/joiner.cpp#L282

**Reply by Leo2442926161** [reply]
Thanks again Jwhui
